cmd/allocator/main.go

```diff
@@ -320,7 +321,17 @@ func runMux(h *serviceHandler, httpPort int) {
 	grpcServer := grpc.NewServer(h.getMuxServerOptions()...)
 	pb.RegisterAllocationServiceServer(grpcServer, h)
 
-	mux := gw_runtime.NewServeMux()
+	mux := gw_runtime.NewServeMux(gw_runtime.WithMarshalerOption(gw_runtime.MIMEWildcard, &gw_runtime.HTTPBodyMarshaler{
```

Comment:
🤔 this looks to be duplicated in a few places - does it make sense to define this in one place and replicate it / create a method that outputs this config?

I'm assuming it's not going to change, but just thinking about if it ever did, remembering to change it in all the the places would be tricky to remember.

Reply:
As you say, it would be better to create a common method.
I'll update this PR.

Reply:
Updated the PR and put the method on util/runtime/runtime.go